templates/encounter/forms/navbar.html.twig

```diff
@@ -0,0 +1,63 @@
+{% block navbar %}
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <span class="navbar-brand">
+        {{ encounterDate|text }} {{ (groupEncounter == true) ? "Group Encounter"|xlt : "Encounter"|xlt}} {{ "for"|xlt }} {{ patientName|text }}
+    </span>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mx-auto">
+            {% for category, details in menuArray %}
+                {% if details.children is iterable %}
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="category_{{ category|xla }}" role="button" data-toggle="dropdown" aria-expanded="false">
+                            {{ category }}
+                        </a>
+                        <div class="dropdown-menu" aria-labelledby="category_{{ category|xla }}">
+                            {% for item in details.children %}
+                                {% set formURL = rootdir ~ "/patient_file/encounter/load_form.php?formname=" ~ item.directory|url_encode %}
+                                <a href="#" class="dropdown-item" onclick="openNewForm('{{ formURL }}', {{ item.displayText|xlFormTitle|attr_js }})">{{ item.displayText|xlFormTitle|text }}</a>
+                            {% endfor %}
+                        </div>
+                    </li>
+                {% else %}
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">{{ category }}</a>
+                    </li>
+                {% endif %}
+            {% endfor %}
+            {% for category, details in moduleMenuArray %}
+                {% if details.children is iterable %}
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" href="#" id="category_{{ category|xla }}" role="button" data-toggle="dropdown" aria-expanded="false">
+                            {{ category }}
+                        </a>
+                        <div class="dropdown-menu" aria-labelledby="category_{{ category|xla }}">
+                            {% for item in details.children %}
+                                <a href="#" class="dropdown-item" onclick="openNewForm('{{ item.href }}', {{ item.menu_name|xlFormTitle|attr_js }})">{{ item.menu_name|xlFormTitle|text }}</a>
```

Comment:
for item.href above, would remove the surrounding single quotes and pipe it through attr_js